pages/index.tsx

```diff
@@ -47,10 +61,10 @@ const Home: FC = () => {
         <Image
           src={star}
           alt="star_img"
-          className="absolute bottom-0 left-[7em] w-8"
+          className="absolute bottom-0 left-[26em] w-8"
         />
         <Image src={star} alt="star_img" className="absolute top-0 right-80" />
-        <h1 className="">
+        <h1 className={`text-[65px] font-bold ${noto_serif.className}`}>
```

Comment:
this also in tailwind.config, use `font-noto`
also add playfair and open sans to tailwind config

Reply:
so should i remove the font.ts file?